cli/fmt.rs

```diff
@@ -167,9 +161,13 @@ fn format_stdin(check: bool) -> Result<(), ErrBox> {
     return Err(OpError::other("Failed to read from stdin".to_string()).into());
   }
   let config = get_config();
+  let formatter = dprint::Formatter::new(&config);
 
-  match dprint::format_text("_stdin.ts", &source, &config) {
-    Ok(None) => unreachable!(),
+  match formatter.format_text("_stdin.ts", &source) {
+    Ok(None) => {
+      // code had a dprint-ignore-file comment
+      stdout().write_all(source.as_bytes())?;
+    },
```

Comment:
The reason this is done is a micro-optimization in wasm to not have to bother sending back the string. I think I should change the api here to only return the string when the file has changed. Thoughts?